src/components/todo/list.js

```diff
@@ -9,8 +10,18 @@ const TodoList = (props)=>{
           className={`complete-${item.complete.toString()}`}
           key={item._id}
         >
+
           <span onClick={() => props.handleComplete(item._id)}>
-            {item.text} : {item.assignee}
+
+            <section className="assignee">
+              {item.assignee}
+            </section>
+            <section className="deleteButton">
+              <button onClick={() => props.deleteItem(item._id)}>X</button>
+            </section>
+            <section>
+              {item.text}
+            </section>
           </span>
         </li>
       ))}
```

Comment:
I am seeing that there is an issue when I try and add an item. That it can't access the item before everything gets initialized. When you have time to come back to this I will regrade for you.